chunky/src/java/se/llbit/chunky/renderer/RenderContext.java

```diff
@@ -68,48 +68,51 @@ public int numRenderThreads() {
   /**
    * @return Scene description file
    */
-  public File getSceneDescriptionFile(String sceneName) {
-    return getSceneFile(sceneName + Scene.EXTENSION);
+  public File getSceneDescriptionFile(String path, String descriptionFileName) {
```

Comment:
Instead of adding the `path` to every method, why not add it as a field to the renderContext and keep the old API?